cvmfs/shortstring.h

```diff
@@ -175,15 +176,34 @@ class ShortString {
     return ShortString(this->GetChars() + start_at, length-start_at);
   }
 
-  static uint64_t num_instances() { return atomic_read64(&num_instances_); }
-  static uint64_t num_overflows() { return atomic_read64(&num_overflows_); }
+  static void RegisterCounters(perf::Statistics *statistics) {
```

Comment:
Hm, that's not too nice yet.  If one forgets to use `RegisterCounters`, it will result in a crash.  I'll have to think about it.